optimas/explorations/base.py

```diff
@@ -147,6 +148,13 @@ def run(self, n_evals: Optional[int] = None) -> None:
             run until the number of evaluations reaches `max_evals`.
 
         """
+        # Store current working directory. It has been observed that sometimes
+        # (especially when using `local_threading`) the working directory
+        # is changed to the exploration directory after the call to `libE`.
+        # As a workaround, the cwd is stored and then set again at the end of
+        # `run`.
+        cwd = os.getcwd()
```

Comment:
Should we bring this up to the `libEnsemble` team?

Reply:
Good point. I opened a new issue https://github.com/Libensemble/libensemble/issues/1244